app/presenters/buyers/account_plans_presenter.rb

```diff
@@ -2,9 +2,10 @@
 
 class Buyers::AccountPlansPresenter < PlansBasePresenter
   def initialize(collection:, params: {})
-    super(service: nil, collection: collection, params: params)
+    super(collection: collection, params: params)
   end
 
+  # This smells of :reek:NilCheck because it is nil check.
```

Comment:
What are these kind of comments about? did you write them manually?

Reply:
Yes. The way reek is disabled/ignored is adding `:reek:` on top of it. I think it's more informative this way.